src/paperoni/discovery/miniconf.py

```diff
@@ -259,16 +259,18 @@ def query(
         base_url = f"https://{base_url}"
 
         url = f"{base_url}/static/virtual/data/{conference}-{year}-orals-posters.json"
-        data = config.fetch.read(
-            url,
-            format="json",
-            cache_into=cache
+        cache_path = (
+            cache
             and config.cache_path
-            and config.cache_path / "miniconf" / f"{conference}-{year}.json",
+            and config.cache_path / "miniconf" / f"{conference}-{year}.json"
         )
+        data = config.fetch.read(url, format="json", cache_into=cache_path)
```

Comment:
Might still be worth it to recheck after a few weeks particularly during the period of the conference? I could believe that they might want to update it to correct some mistakes if they notice any? A bit less efficient but I suppose we can live with a a few downloads for each conference through the year?

Reply:
I added the parameter